internal/docstore/docstore.go

```diff
@@ -52,7 +54,21 @@ var NewCollection = newCollection
 
 // newCollection makes a Collection.
 func newCollection(d driver.Collection) *Collection {
-	return &Collection{driver: d}
+	c := &Collection{driver: d}
+	_, file, lineno, ok := runtime.Caller(1)
+	runtime.SetFinalizer(c, func(c *Collection) {
```

Comment:
Do you want to check this in? We don't have it for other top-level structs like `blob.Bucket` (just for smaller-scoped things like `blob.Reader`). But maybe we should...?

Reply:
In a later CL I'll make a note to remove it. We'll be adding more tests, so it may still be useful.